stacks-core/src/address.rs

```diff
@@ -35,6 +32,40 @@ pub enum AddressVersion {
     TestnetMultiSig = 21,
 }
 
+/// Supported stacks address versions
+#[repr(u8)]
+#[derive(FromRepr, EnumIter, EnumString, PartialEq, Eq, Copy, Clone, Debug)]
+#[strum(ascii_case_insensitive)]
+pub enum AddressKind {
+    /// Pay to public key hash
+    P2PKH = 0,
+    /// Pay to script hash
+    P2SH = 1,
+    /// Pay to witness public key hash
```

Comment:
Maybe add a comment that this is the only AddressKind used in Stacks for single accounts

Reply:
It's actually a p2pkh if you look at the `Address::from_public_key` function. I'm not sure at the moment which one should it be.

Reply:
p2pkh looks correct

Reply:
I removed AddressKind completely. It's probably a remnant of the code I used to signal what kind of Stacks address should be used to burn from. We now by convention always use P2PKH for that.